### ISSUE TITLE
Create mid-project release plan

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#62]
## User story
1. As POs
2. We want to create a mid-project release plan
3. So that we can track all features and their size. 

## Acceptance criteria
* Included in the mid-project release section for the planning documents
* Burn-down chart displays correctly.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
